app/main/authorize/keycloak_login_required_decorator.py

```diff
@@ -37,30 +37,23 @@ def protected_route():
     @wraps(view_func)
     def decorated_view(*args, **kwargs):
         if current_app.config["TESTING"] and not current_app.config.get(
-            "FORCE_AUTHENTICATION_FOR_IN_TESTING"
+                "FORCE_AUTHENTICATION_FOR_IN_TESTING"
```

Comment:
did we have a change in our formatting configuration causing these indentation diff?

Reply:
so it seems the pre-commit github action check is failing due to formatting issues.

If you haven't already, could you  run `pre-commit install` in your poetry environment, to ensure that every time you commit it runs the pre-commit hooks.

To fix all the files see which ones are causing issues in the gh action, and then manually run `pre-commit run --files LIST_OF_FILES_TO_RUN_ON`, and then commit them (the checks should run against all staged files but pass now that the first run fixed them)